CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 ## envconsul CHANGELOG
 
+## v0.9.2 (Jan XX, 2020)
+
 SECURITY:
 
-* Don't log secret values pulled from vault.
+* Don't log values/secrets pulled from vault [[GH-226](https://github.com/hashicorp/envconsul/pull/226)]
+
+BUG FIXES:
+
+* Build Arm binaries with CGO enabled [[GH-227](https://github.com/hashicorp/envconsul/issues/227)]
```

Comment:
random q: trying to find the pattern - I was wondering when to link a GitHub issue vs. PR?

Reply:
I've been playing this by ear (it wasn't consistent before and they didn't include links) but have mostly settled on linking to both with the issue first and the PR second. Which means I missed the PR here. Thanks!